build/_output/bin/kabanero-rest-services/.gitignore

```diff
@@ -0,0 +1 @@
+/main
```

Comment:
this file doesn't seem like it should be checked in....

Reply:
ok